modules/nativescript/nativescript.cpp

```diff
@@ -407,6 +412,24 @@ Variant NativeScript::_new(const Variant **p_args, int p_argcount, Variant::Call
 		return Variant();
 	}
 
+	call("_init", p_args, p_argcount, r_error);
+
+	if (r_error.error != Variant::CallError::CALL_OK) {
```

Comment:
hmm, but it fails for me. "_init" is defined in object without any default implementation - are you sure it should work?